baseimages/scanner/cli/scan.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package cli
+
+import (
+	"context"
+	"encoding/json"
+
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/Azure/acr-builder/baseimages/scanner/scan"
+	"github.com/Azure/acr-builder/cmder"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+const scanLongDesc = `
+This command can be used to scan a Dockerifle.
+`
+
+type scanCmd struct {
+	out               io.Writer
+	dockerfile        string
+	context           string
+	tags              []string
+	buildArgs         []string
```

Comment:
>buildArgs [](start = 1, length = 9)

is the secret build args relevant here?

Reply:
not really, they just resolved and substituted as normal args to be used when scanning the Dockerfile